### ISSUE TITLE
Archive Podcast Shows

### ISSUE THREAD

**Issue by emkay5771** [Automattic/pocket-casts-android#796]
### What problem does this address?

Currently, when a show goes on extended hiatus (or potentially ends), you need to keep the podcast displayed on the home screen to be notified if it does come back.

### What is your proposed solution?

It would be awesome if podcasts could be archived (similar to the behavior of individual episodes), which would hide the podcast from view unless a new episode is posted. It would enable users to clean up their home screen while not missing out on the return of a podcast!

### Did you search for existing feature suggestions?

- [X] I have searched for existing feature suggestions.

**Comment by CookieyedCodes** [comment]
I personally would prefer a smart folder which would automatically adds feeds to that folder after a set hiatus, I would also set this up to pickup on feeds that are completely marked as played with the option to hide that folder in settings (obviously ignore the podcast if it's allready in a folder tho) might be a good demo for the folder function ie the user can't add feeds to it themselves 🤔 😉, at that their are other things that I would prefer the app has first but hopefully someday🤔 😉

**Comment by twizzlebizzle** [comment]
Yeah, I've moved them all into a folder which seems a less elegant solution

**Comment by mohi-TOS** [comment]
There could also be a feature to fade out podcasts with no active episodes.

**Comment by CookieyedCodes** [comment]
> There could also be a feature to fade out podcasts with no active episodes.

I wouldn't mind that regardless but it dosen't really clean stuff up in my opinion 🤔 😅
